src/opm/parser/eclipse/EclipseState/Aquancon.cpp

```diff
@@ -80,20 +81,36 @@ namespace Opm {
                 for (int j=m_aqurecord.at(aquanconRecordIdx).j1; j <= m_aqurecord.at(aquanconRecordIdx).j2; j++)
                     for (int i=m_aqurecord.at(aquanconRecordIdx).i1; i <= m_aqurecord.at(aquanconRecordIdx).i2; i++)
                         m_aqurecord.at(aquanconRecordIdx).global_index_per_record.push_back
-                                                            (grid.getGlobalIndex(i-1 ,j-1 ,k- 1)
+                                                            (
+                                                                grid.getGlobalIndex(i-1, j-1, k-1)
                                                             );
             }
             size_t global_index_per_record_size = m_aqurecord.at(aquanconRecordIdx).global_index_per_record.size();
-            m_aqurecord.at(aquanconRecordIdx).influx_coeff_per_record.resize(global_index_per_record_size,influx_coeff);
+
+            m_aqurecord.at(aquanconRecordIdx).influx_coeff_per_record.resize(global_index_per_record_size, nullptr);
+
+            if (aquanconRecord.getItem("INFLUX_COEFF").hasValue(0))
+            {
+                double* influx_coeff = new double( aquanconRecord.getItem("INFLUX_COEFF").getSIDouble(0) );
```

Comment:
That's a guaranteed memory leak.  There's nothing that will release the resources acquired by `influx_coeff` here.  Is there any reason not to make `influx_coeff` a regular `double`?

Reply:
@bska, true. Shall I write a destructor which explicitly deletes them? This change came about after a rather lengthy discussion about the default value behaviour of the aquifer initial pressure p0 (https://github.com/OPM/opm-common/pull/476).

Reply:
I am all for using doubles here if it is not already clear from the previous discussions...

Reply:
Oh weak of me. I could just do this instead, right?

double influx_coeff = aquanconRecord.getItem("INFLUX_COEFF").getSIDouble(0);

And then I continue on with the modifications suggested by atgeirr.?

Reply:
Yes! Or even better: `const double influx_coeff = ...`.